spec/src/main/asciidoc/WebServicesMetadata.adoc

```diff
@@ -943,2259 +807,1399 @@ header, and the mode is OUT.
+This section defines a standard mapping from a service endpoint
+interface or service implementation bean to the SOAP 1.1 binding.
+Implementers MAY also support other bindings, but these bindings are
+non-standard. If Jakarta Web Services Metadata implementation supports bindings other than
+SOAP 1.1, it MUST include a mechanism to selectively enable or disable
+these bindings.
 
-a|
-@WebResult.name
+By default Jakarta Web Services Metadata follows the SOAP binding defined in Jakarta XML Web Services 2.3 [5],
+section 10.
 
-|===
+=== Operation Modes
 
+Jakarta Web Services Metadata implementations are REQUIRED to support the following WS-I
+compliant operation modes:
 
-[cols=",,",]
-|===
-| a|
-style and the parameter style is BARE.
+* Operations with the rpc style and literal use (rpc/literal)
+* Operations with the document style and literal use (document/literal).
 
-|
+Implementations MAY optionally support operation modes with the encoded
+use (document or rpc style). The developer MAY indicate which operation
+mode is in effect by specifying the appropriate @SOAPBinding.style and
+@SOAPBinding.use annotations at the class or interface level.
 
-a|
-targetNamespace
+==== RPC Operation Style
 
-a|
-The XML namespace for the return value.
+In the RPC operation style, the parameters and return values map to
+separate parts on the WSDL input and output messages. The @WebParam.mode
+annotation determines the messages in which a particular parameter
+appears. IN parameters appear as parts in the input message, OUT
+parameters appear as parts in the output message, and INOUT parameters
+appear as parts in both messages. The order of parameters in the method
+signature determines the order of the parts in the input and output
+message. The return value is the first part in the output message.
 
-a|
-The empty namespace, if the operation is document style, the
+In the rpc/literal operation mode, each message part refers to a
+concrete schema type. The schema type is derived from the Java type for
+the parameter, as described in section 5 - Java Mapping To XML/WSDL.
 
-| a|
-Only used if the operation is
+==== Document Operation Style
 
-a|
-parameter style is
+In the document operation style, the input and output WSDL messages have
+a single part referencing a schema element that defines the entire body.
+Jakarta Web Services Metadata implementations MUST support both the “wrapped” and “bare”
+styles of document / literal operation. The developer may specify which
+of these styles is in effect for a particular operation by using the
+@SOAPBinding.parameterStyle annotation.
 
-| a|
-document style or the return value
+==== Document “Wrapped” Style
 
-a|
-WRAPPED, and the
+In the “wrapped” operation style, the input and output messages contain
+a single part which refers (through the _element_ attribute) to a global
+element declaration (the _wrapper_)
 
-| a|
-maps to a header.
+of complexType defined using the xsd:sequence compositor. The global
```

Comment:
no need for extra empty lines here, just continue the sentence...

Reply:
ok, reslved by https://github.com/eclipse-ee4j/jws-api/pull/69/commits/62fdf397f28a148826367da36bcaef2c30e97c7d